src/scripts/test.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+set -euxo pipefail
+cd "$(dirname "${BASH_SOURCE[0]}")/../.."
+
+export ORY_SDK_URL=https://playground.projects.oryapis.com
+export ORY_UI_URL=https://playground.projects.oryapis.com/ui
+
+#
+# Please add any build steps to the Makefile and not here!
+#
+
+cd django-ory-cloud && \
+  cd mysite && \
+  poetry run python manage.py migrate && \
+poetry run python manage.py runserver &
+ory proxy --no-jwt --port 4000 http://localhost:8000/ &
```

Comment:
By default, Bash exits a script when a command fails. So this should work similar to your code:

```suggestion
cd django-ory-cloud/mysite
poetry run python manage.py migrate
poetry run python manage.py runserver &
ory proxy --no-jwt --port 4000 http://localhost:8000/ &
```